internal/versions/versions.go

```diff
@@ -307,32 +307,41 @@ func (h *handler) GetLatestOsImage(cpuArchitecture string) (*models.OsImage, err
 	return latest, nil
 }
 
-func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion, cpuArchitecture string) (*models.ReleaseImage, error) {
+func (h *handler) AddReleaseImage(releaseImageUrl, pullSecret, ocpReleaseVersion string, cpuArchitectures []string) (*models.ReleaseImage, error) {
 	var err error
-	// If release version or cpu architecture are not specified, use oc to fetch values
-	if ocpReleaseVersion == "" || cpuArchitecture == "" {
+	var cpuArchitecture string
+	var osImage *models.OsImage
+
+	// If release version or cpu architectures are not specified, use oc to fetch values
+	if ocpReleaseVersion == "" || cpuArchitectures == nil {
```

Comment:
cpuArchitectures shouldn't be an empty array, but maybe worth checking as well just in case.

Reply:
That's a good catch, thanks!